### DISCUSSION TITLE
How to debug high CPU usage of the docker

### DISCUSSION THREAD

**Discussion by yarkinwho** [blockscout/blockscout#8671] [Q&A]
I have a blockscout running in docker, and it seems the CPU usage is keep growing. I can confirm it's mainly the indexer as I have two separate docker containers, one with indexer disabled and one with web app disabled and it is clear it's the indexer one using much more CPU. (the CPU usage of the web app one also grows but seems much slower)

My question is, what's the proper way to debug this cpu usage? Or how can I collect information useful for devs to identify the issue? 

I am not really familiar with erlang so please give me more details if possible... 

Thanks.

**Comment by ekawibisono** [comment]
What pc or server specifications are you using to install blockscout?

**Reply by yarkinwho** [reply]
tried on aws 
t3a.large (2core 4G)
t3a.xlarge (4 core 8G)
t3a.2xlarge (8core 16G)

All shows consistent CPU consumption growth.
